gen-ai/orchestrator-server/src/main/python/server/src/gen_ai_orchestrator/services/langchain/rag_chain.py

```diff
@@ -312,3 +337,46 @@ def get_rag_debug_data(
         answer=response['answer'],
         duration=rag_duration,
     )
+
+
+def check_guardrail_output(guardrail_output: dict) -> bool:
+    """Checks if the guardrail detected toxicities.
+
+    Parameters
+    ----------
+    guardrail_output: dict
+
+    Returns
+    -------
+    Returns True if nothing is detected, raises an exception otherwise.
+    """
```

Comment:
garde la même manière de décrire les fonctions, pour homogénéisé le doc
```suggestion
    """
    Checks if the guardrail detected toxicities.
    Args:
        guardrail_output: The guardrail output dictionnary
    Returns:
        Returns True if nothing is detected, raises an exception otherwise.   
    """
```